src/applications/travel-pay/containers/TravelPayStatusApp.jsx

```diff
@@ -346,140 +349,142 @@ export default function TravelPayStatusApp({ children }) {
         <h1 tabIndex="-1" data-testid="header">
           {title}
         </h1>
-        <div className="vads-l-col--12 medium-screen:vads-l-col--8">
-          {smocEnabled ? (
-            <SmocEntryContent />
-          ) : (
-            <>
-              <h2 className="vads-u-font-size--h4">
-                You can use this tool to check the status of your VA travel
-                claims.
-              </h2>
-              {!error &&
-                !isLoading && (
-                  <va-additional-info
-                    class="vads-u-margin-y--3"
-                    trigger="How to manage your claims or get more information"
-                  >
-                    <>
-                      <HelpTextManage />
-                      <va-link
-                        data-testid="status-explainer-link"
-                        href="/my-health/travel-pay/help"
-                        text="What does my claim status mean?"
-                      />
-                    </>
-                  </va-additional-info>
-                )}
-            </>
-          )}
-
-          {isLoading && (
-            <va-loading-indicator
-              label="Loading"
-              message="Loading Travel Claims..."
-            />
-          )}
-          {!isLoading &&
-            data.length > 0 && (
+        <DowntimeWindowAlert appTitle="Travel Pay">
```

Comment:
This wrapper strikes me as something that might be fitting to add at the `App.jsx` level rather than within each individual component. Besides showing titles alongside the downtime alert, is there something this approach gets us that would lead us to avoid adding it at a more global level?

Reply:
Yes, I totally agree and that was my first approach

But

Since how we have the application set up, if we move this to the App then we lose headings and navigation. So in order to move it to the app level  we would need to refactor a bunch of the application